<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_Preamble">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Preamble(7Byte)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_SFD">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>SFD(2Byte)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_TMAC">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TARGET-MAC(6Byte)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_SMAC">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>SOURCE-MAC(6Byte)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_LENGTH">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>LENGTH(2Byte)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_DATA">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>DATA(VAR) --ASCII</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_FCS">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>FCS(4Byte) -- CRC32</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_preamble">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>7d:3f:42:f2:a7:61:12</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sfd">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>a3:1f</string>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_TMAC">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>40:e2:30:58:c6:0d</string>
   </property>
   <property name="maxLength">
    <number>17</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SMAC">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>02:42:a8:01:ae:c8</string>
   </property>
   <property name="maxLength">
    <number>17</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_DATA">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_fcsvalue">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>MakeEthernet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_LengthValue">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
